### ISSUE TITLE
Build for prod still looking for aurelia-testing

### ISSUE THREAD

**Issue by justindra** [aurelia/cli#759]
**I'm submitting a bug report**

* **Library Version:**
0.31.3

**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
6.11.1

* **NPM Version:**
3.10.10

* **Browser:**
Chrome | Firefox | Edge

* **Language:**
TypeScript

* **Loader/bundler:**
all

**Current behavior:**
```
au new hello-world
```

Select all default typescript options

```
cd hello-world
npm install
au build --env prod
http-server .
```

Go to the `localhost:[port]` specified, and it does not load as it 404s missing:
`/aurelia-testing`

* **What is the expected behavior?**
It should not be looking for `aurelia-testing` as in the `aurelia.json` file it has been specified an environment of `dev` only and in `main.ts` it also has an environment flag

```
  if (environment.testing) {
    aurelia.use.plugin('aurelia-testing');
  }
```

* **What is the motivation / use case for changing the behavior?**
To be able to create a production build with the CLI.

As a note, using `au run --env prod` seems to work with no problems at all.

**Comment by JeroenVinke** [comment]
This is strange, i'm not able to reproduce the issue. Just to confirm, you're using the default typescript setup right (requirejs and aurelia-cli bundler). On build (`au build --env prod`) the CLI copies the appropriate environment file from `aurelia_project/environments/prod.ts` to `src/environment.ts`. Can you confirm that this actually happens, so that `src/environment.ts` has `testing` set to false?

**Comment by justindra** [comment]
I updated my node major version to 7 and it seems to work now, even when I drop back to 6 again. Weird.., I will close this for now. Thanks @JeroenVinke

**Comment by justindra** [comment]
Just wanted to give an update as I found out what the issue was.

Basically the bundled files was referring to other bundled files without the hash and as aurelia-cli does not clear the scripts folder beforehand, on serving it, it found an old version of `app.js` without the hash, which was created during the Dev build. Obviously that was still referencing Aurelia-testing.

This issue has been resolved in the latest version of the cli. Although it's still not removing the previous bundles in scripts folder.
